helm/cluster-azure/values.yaml

```diff
@@ -1,86 +1,57 @@
-metadata:
-  # Cluster name. Defaults to chart release name
-  name: ""
-
-  # Cluster description used in metadata.
-  description: "test"
-
-  # Organization in which to create the cluster.
-  organization: ""
-
-providerSpecific:
-  location: "westeurope"
-  # This value must be updated to the right value for the cluster to be installed
-  subscriptionId: PLACEHOLDER
-  azureClusterIdentity:
-    name: "cluster-identity"
-    namespace: "org-giantswarm"
-
 connectivity:
   bastion:
     enabled: true
     instanceType: Standard_D2s_v5
   network:
-    serviceCidr: "172.31.0.0/16"
-    podCidr: "192.168.0.0/16"
-    hostCidr: "10.0.0.0/16"
     controlPlane:
       cidr: 10.0.0.0/20
+    hostCidr: 10.0.0.0/16
+    mode: public
+    podCidr: 192.168.0.0/16
+    serviceCidr: 172.31.0.0/16
     workers:
       cidr: 10.0.16.0/20
-    mode: public
-  sshSSOPublicKey: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIM4cvZ01fLmO9cJbWUj7sfF+NhECgy+Cl0bazSrZX7sU vault-ca@vault.operations.giantswarm.io"
-
+  sshSSOPublicKey:
+    ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIM4cvZ01fLmO9cJbWUj7sfF+NhECgy+Cl0bazSrZX7sU
+    vault-ca@vault.operations.giantswarm.io
```

Comment:
I feel that this rendering is slightly out of the ordinary. While I'm sure it's valid, I wonder how many will stumble upon this and doubt the validity.

It's a long string that gets wrapped and then the wrapped line gets an additional indentation.

Do we have any influence on this?

Reply:
That is not `helm-values-gen`. That is just me formatting with prettier so that I can see how the diffs get displayed in the output. So no worries, it will be gone in a minute.

Reply:
Ok, forget what I said, this actually is a problem.

I will investigate this.

Reply:
Fixed in https://github.com/giantswarm/helm-values-gen/pull/3